src/values/UnDeserializableValue.js

```diff
@@ -117,8 +122,11 @@ var SELF = dv.UnDeserializableValue = util.inherit(
 	 * @throws {Error} whenever the function is called (implementation missing).
 	 */
 	toJSON: function() {
-		// TODO
-		throw new Error( 'Not implemented yet.' );
+		return {
+			type: this.getTargetType(),
+			value: this.getStructure(),
+			error: this.getReason()
```

Comment:
The order in the PHP class is `value`, `type`, `error`. Probably irrelevant at this point since JSON does not guarantee an order anyway.